setup.py

```diff
@@ -67,4 +67,6 @@ def read(x):
       zip_safe=False,
       entry_points={
           "console_scripts": ['zodbupdate = zodbupdate.main:main']
-      })
+      },
+      test_suite="zodbupdate.tests.test_suite",
```

Comment:
I do not like this as it adds another way to run the tests.

Reply:
Since this is package so low-level, it simplifies tst calls - no zope.testrunner needed here.